test/certs/server-ecdsa-brainpoolP256r1-cert.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think we have a script to generate those files. Can you update that?

Reply:
Done